app/src/main/java/mozilla/lockbox/store/DataStore.kt

```diff
@@ -261,21 +253,15 @@ open class DataStore(
             .map {
                 log.debug("Hashed UID: $it")
             }
-            .timeout(Constant.App.syncTimeout, TimeUnit.SECONDS)
-            .doOnEvent { _, err ->
-                (err as? TimeoutException).let {
-                    // syncStateSubject.accept(SyncState.TimedOut)
-                    dispatcher.dispatch(DataStoreAction.SyncTimeout(it?.message ?: ""))
-                }.run {
-                    syncStateSubject.accept(SyncState.NotSyncing)
-                }
+            .doOnEvent { _, _ ->
```

Comment:
Can we remove 

```kotlin
            .map {
                log.debug("Hashed UID: $it")
            }
```

too?

Reply:
That's going to be handled in #870 (next up in the sprint backlog! 🎸)